src/data/faq.json

```diff
@@ -2703,7 +2703,7 @@
                 {
                     "term": "Effective validity",
                     "anchor": "effective_validity",
-                    "description": "We refer to the duration of vaccination protection as 'effective validity'. The effective validity of the vaccination thus counts from the last vaccination date. The effective validity of the vaccination protection may vary from country to country. According to the <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Genesenennachweis.html' target='_blank' rel='noopener noreferrer'> RKI </a>, the maximum effective validity of a [recovery certificate](https://www.coronawarn.app/en/faq/#glossary_recovery_certificate) is 90 days."
+                    "description": "We refer to the duration of vaccination protection as 'effective validity'. The effective validity of the vaccination thus counts from the last vaccination date. The effective validity of the vaccination protection may vary from country to country. According to the <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Genesenennachweis.html' target='_blank' rel='noopener noreferrer'> RKI </a>, the maximum effective validity of a <a href='https://www.coronawarn.app/en/faq/#glossary_recovery_certificate' target='_blank' rel='noopener noreferrer'>recovery certificate</a> is 90 days."

```

Comment:
It would be better to use a relative link.
`rel='noopener noreferrer'` is not necessary for references a location on the same web and can be removed.

So this would be better and shorter:
`<a href='#glossary_recovery_certificate' target='_blank'>recovery certificate</a>`

This is the way the link is done in faq_de.json.

Reply:
Done